src/main/java/games/strategy/engine/data/ResourceCollection.java

```diff
@@ -121,18 +121,12 @@ private void subtract(final ResourceCollection resourceCollection) {
     subtract(resourceCollection.m_resources);
   }
 
-  public void subtract(final IntegerMap<Resource> cost) {
+  private void subtract(final IntegerMap<Resource> cost) {
     for (final Resource resource : cost.keySet()) {
       removeResource(resource, cost.getInt(resource));
     }
   }
 
-  public void subtract(final IntegerMap<Resource> cost, final int quantity) {
```

Comment:
I couldn't find any reflection references using similar search parameters as above.

There are direct calls to the other `ResourceCollection#subtract()` overloads; no one is apparently calling this particular one.

Searching through the history, this overload was added in 2120a28.  Searching all commits since that one that contain `subtract(` shows that none of them were ever calling this overload.  So possibly a YAGNI.